cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraDaemonProcess.java

```diff
@@ -431,10 +431,17 @@ public void drain()
      * @throws ExecutionException   If execution is interrupted.
      * @throws IOException          If communication with Cassandra fails.
      */
-    public void upgradeTables()
+    public void upgradeSSTables(String keyspace, List<String> columnFamilies)
             throws InterruptedException, ExecutionException, IOException {
-        for (String keyspace : getNonSystemKeySpaces()) {
-            getProbe().forceKeyspaceCleanup(0, keyspace);
-        }
+        String[] families = new String[columnFamilies.size()];
```

Comment:
Yeah it is already initialized appropriately: 
`this.columnFamilies = (columnFamilies == null) ?
                Collections.emptyList() :
                columnFamilies;`
in UpgradeSSTableContext.java